benchmarks/example.cpp

```diff
@@ -0,0 +1,22 @@
+#include <benchmark/benchmark.h>
+#include <pcl/features/normal_3d.h> // for NormalEstimation
+#include <pcl/io/pcd_io.h> // for PCDReader
+
+static void BM_NormalEstimation(benchmark::State& state) {
+  // Perform setup here
+  pcl::PointCloud<pcl::PointXYZ>::Ptr cloud (new pcl::PointCloud<pcl::PointXYZ>);
+  pcl::PCDReader reader;
+  reader.read ("/home/markus/pcd_files/milk_cartoon_all_small_clorox.pcd", *cloud);
```

Comment:
This needs to be parsed like with the tests as runtime argument? Or all test has to be run on generated clouds.